.github/workflows/format.yml

```diff
@@ -0,0 +1,20 @@
+name: Formatting
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js 18
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+        cache: 'npm'
+    - run: npm ci

```

Comment:
What does `npm ci` do?

Reply:
Was getting a bit confused over if I should be running `npm ci` or `npm install`. Been reading up on it and I think the fastest setup for us is first caching the `node_modules/` then running `npm install`, which grabs the dependencies if they're not cached.